Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -4,19 +4,47 @@
 
 namespace BootCamp.Chapter
 {
+    // ToDo: Change code so it only allows letters from an alphabet.
+
     /// <summary>
     /// Caesar cipher which supports all ASCII character (256 in total)
     /// </summary>
     public static class CaesarCipher
     {
+        private const char UpperA = 'A';
```

Comment:
This variable name is absolutely redundant. It still doesn't explain what the a's are used for.

Reply:
Yes, that makes sense! I'll keep this in mind! :)